dbt/include/global_project/macros/materializations/incremental/bigquery_incremental.sql

```diff
@@ -0,0 +1,58 @@
+
+{% materialization incremental, adapter='bigquery' -%}
+
+  {%- set unique_key = config.get('unique_key') -%}
+  {%- set sql_where = config.get('sql_where') -%}
+
+  {%- set non_destructive_mode = (flags.NON_DESTRUCTIVE == True) -%}
+  {%- set full_refresh_mode = (flags.FULL_REFRESH == True) -%}
+
+  {% if non_destructive_mode %}
+    {{ log("--non-destructive is not supported on BigQuery, and will be ignored", info=True) }}
```

Comment:
It seems to me like this should fail with an error rather than just logging and ignoring it.